sdks/java/core/src/test/java/org/apache/beam/sdk/io/WriteTest.java

```diff
@@ -295,9 +294,8 @@ public void populateDisplayData(DisplayData.Builder builder) {
   @Test
   public void testWriteUnbounded() {
     TestPipeline p = TestPipeline.create();
-    PCollection<String> unbounded = PCollection.<String>createPrimitiveOutputInternal(p,
-        WindowingStrategy.globalDefault(),
-        IsBounded.UNBOUNDED).setCoder(StringUtf8Coder.of());
+    PCollection<String> unbounded =
+        p.apply(Create.of("foo")).setIsBoundedInternal(IsBounded.UNBOUNDED);
```

Comment:
streaming as true doesn't change the output of create;

`UnboundedReadFromBoundedSource` is in runners-core (so isn't visible) and produces an `IsBounded.BOUNDED PCollection` as the output of the composite.